spanner/google/cloud/spanner/database.py

```diff
@@ -389,8 +389,7 @@ def batch(self):
         """
         return BatchCheckout(self)
 
-    def snapshot(self, read_timestamp=None, min_read_timestamp=None,
-                 max_staleness=None, exact_staleness=None):
+    def snapshot(self, **kw):
```

Comment:
@jonparrott I see no value in "human linting" a style point which is completely debatable, where the Python community / stdlib is also divided.  If / when we re-enable `pylint`, and it *does* complain, then we fix *all* points in a single "hygeine" PR.  Until then, it is *de gustibus no disputandum est.*